lib/OpenQA/Utils.pm

```diff
@@ -429,6 +432,13 @@ sub find_labels {
     return \@labels;
 }
 
+sub find_flags {
+    my $text = shift // '';
```

Comment:
Just wondering if you can use a signature here? If not ignore me. I don't remember if this would need more changes here.

```suggestion
sub find_flags($text) {
```

Reply:
please use signatures here